### ISSUE TITLE
Tab Menu: Issue Title

### ISSUE THREAD

**Issue by harrisonwillettbetterworks** [primefaces/primevue#3583]
### Describe the bug
The Tab Menu component has a WCAG SC 1.3.1 failure found here:
https://github.com/primefaces/primevue/blob/b23b31461c27e3a3ac69d7272e3914206479c3ae/components/tabmenu/TabMenu.vue#L37

When running Axe Dev Tools on any Tab Menu two errors are seen both sighting a failure of SC 1.3.1. By adding the `role="presentation"` to the inkbar this error should be resolved.
 
 ### Proposed Fix
 `<li role="presentation" ref="inkbar" class="p-tabmenu-ink-bar"></li> `

### PrimeVue version
3.22.4

### Vue version
3.x

### Language
TypeScript
